packages/react/src/components/TreeView/TreeView.tsx

```diff
@@ -119,8 +195,9 @@ export default function TreeView({
     if (match(event, keys.ArrowDown)) {
       nextFocusNode = treeWalker.current.nextNode();
     }
+    // @ts-ignore
```

Comment:
What error is being suppressed here?

Reply:
The error here is the same as in the previous comment
![image](https://github.com/carbon-design-system/carbon/assets/25573926/2f4130ca-928c-4431-afe2-aca3cdb65f46)